main.py

```diff
@@ -1,10 +1,45 @@
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
+import json
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for name in players:
+        nickname = name
+        email = players[name]["email"]
+        bio = players[name]["bio"]
+
+        race = players[name]["race"]
+        race_name = race["name"]
+        race_description = race["description"]
+        race_inst = Race.objects.get_or_create(
+            name=race_name, description=race_description
+        )
```

Comment:
The `get_or_create` method returns a tuple (instance, created). Ensure you are using `race_inst[0]` correctly to access the Race instance.